spaceship.zsh

```diff
@@ -210,6 +215,26 @@ spaceship_ruby_version() {
   echo -n "%{$reset_color%}"
 }
 
+# Show current vi_mode mode
+spaceship_vi_mode() {
+  if [[ $(bindkey | grep "vi-quoted-insert") ]]; then # check if vi-mode enabled
+    echo -n "%{$fg_bold[gray]%}"
+
+    MODE_INDICATOR="${SPACESHIP_VI_MODE_INSERT}"
+
+    case ${KEYMAP} in
+      main|viins)
+        MODE_INDICATOR="${SPACESHIP_VI_MODE_INSERT}"
+        ;;
+      vicmd)
+        MODE_INDICATOR="${SPACESHIP_VI_MODE_NORMAL}"
+        ;;
+    esac
+    echo -n "${MODE_INDICATOR}"
```

Comment:
Would be great if the indicator will be highlighted as **white bold**. What do you think?